heron_wsgi/admin_lib/ldaplib.py

```diff
@@ -288,17 +288,17 @@ def _script():  # pragma nocover
         import ldap
 
         logging.basicConfig(level=logging.INFO)
+        cwd = Path('.', open=io_open, joinpath=joinpath, exists=exists)
```

Comment:
I still don't see why you're touching `ldaplib`. Clue me in?

Reply:
the heron_policy jenkins job runs it
https://yellowfin:8443/job/heron_policy/configure